### DISCUSSION TITLE
`except_` and similar cause unnecessary and unwanted column name prefixing.

### DISCUSSION THREAD

**Discussion by billytetrud** [sqlalchemy/sqlalchemy#8817] [Usage Questions]
### Describe the bug

`Query.except_` and similar constructs (like `union_`) cause unnecessary query wrapping and column name prefixing that can't be removed or changed back by any means I can figure out.

There's a few questions on stack overflow:
* [This one](https://stackoverflow.com/questions/60318564/sql-alchemy-prefixes-table-name-to-columns/73419585#73419585) has an answer I wrote, but it requires not using `except_`.
* [This one](https://stackoverflow.com/questions/73588253/how-do-you-prevent-sqlalchemy-from-prefixing-column-names-when-using-except) that I asked that has no answer.

### To Reproduce

```python
`Model.query.except_(Model.query).subquery().c.some_column_name`
```

The above won't work ^ because the column name will be prefixed (eg like `models_some_column_name`).

### Error

No error.

### Versions

- OS: Ubuntu 22
- Python: 3.6.9
- SQLAlchemy: 1.1.1
- Database: Postgres
- DBAPI (eg: psycopg, cx_oracle, mysqlclient): Not sure.

**Comment by zzzeek** [comment]
I agree totally. that's why [Query is now legacy](https://docs.sqlalchemy.org/en/14/changelog/migration_20.html#orm-query-unified-with-core-select) - the awkwardness of unions and set operations was one of the primary motivators.

In SQLAlchemy 1.4, use the select() construct directly:

```py
class A(Base):
    __tablename__ = "a"

    id = Column(Integer, primary_key=True)
    data = Column(String)


except_stmt = select(A).except_(select(A))

# there's the statement
print(except_stmt)

# to get A entities from a set operation, select the entity then
# apply the query you want to from_statement
stmt = select(A).from_statement(except_stmt)

print(stmt)
```

docs at https://docs.sqlalchemy.org/en/14/orm/queryguide.html#selecting-entities-from-unions-and-other-set-operations

**Reply by billytetrud** [reply]
Ah I see, thanks for the tip! Would be nice to upgrade to a modern version. Finger crossed.
